### ISSUE TITLE
How to test c++ code or binary

### ISSUE THREAD

**Issue by Fayozbek** [hawkeyesec/scanner-cli#138]
# Issue Template

### Description

Thank you for your awesome project.
1. I want to test C++ code or binaries by this fuzzer. maybe lack of my knowledge  I could not find any usage for c++ code.  
2. Hawkeye can test binaries which are written in C++? If the answer is YES can you share usage for this?
Appreciate it

### Version

Latest version

**Comment by felixhammerl** [comment]
Fuzzing is not in my area of expertise, neither is C++, and there's nothing to that end in Hawkeye, but @bekh6ex  has some experience with fuzzing. I feel this is slightly out of scope for this project, but Aleksey can probably point out where you would have to start looking ...

**Comment by Fayozbek** [comment]
Yeah, I think there was a misunderstanding from my side. I thought "Hawkeye: Towards a Desired Directed Grey-box Fuzzer" - "https://chenbihuan.github.io/paper/ccs18-chen-hawkeye.pdf"
Anyway, thank you for your kind reply!
